guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -52,10 +55,7 @@ If you want to change the default values in the template, clone the template and
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-{foreman-installer} --foreman-proxy-dhcp-ipxefilename "http://{foreman-example-com}/unattended/iPXE?bootstrap=1"
+{foreman-installer} --foreman-proxy-dhcp-ipxefilename "http://_{foreman-example-com}_/unattended/iPXE?bootstrap=1"
 ----
 +
-[NOTE]
-====
-Ensure that `--foreman-trusted-proxies` is configured correctly on the {ProjectServer}.
-====
\ No newline at end of file
+If you want to use {SmartProxyServer} instead, run this command on {SmartProxyServer} using the {SmartProxy}'s hostname in the URL.
```

Comment:
I don't quite understand this.

Reply:
This is supposed to be for the case when the user has external SmartProxy and wants to use it for iPXE booting instead of the ProjectServer directly. I've tried to improve it.
Also, it must be verified by a SME :) I'm not 100% sure if I'm gettting it right.